scripts/stac/imagery/tests/conftest.py

```diff
@@ -21,3 +21,8 @@ def fake_collection_metadata() -> Iterator[CollectionMetadata]:
         "geographic_description": None,
     }
     yield collection_metadata
+
+
+@pytest.fixture(autouse=True)
```

Comment:
I don't think `autouse=True` [is necessary](https://docs.pytest.org/en/6.2.x/fixture.html#autouse-fixtures-fixtures-you-don-t-have-to-request)

Reply:
Interesting, it also appears to be safe to remove from L10.